components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/OAuthAdminServiceImpl.java

```diff
@@ -1406,9 +1406,9 @@ public void updateOauthSecretKey(String consumerKey) throws IdentityOAuthAdminEx
 
     private String getLoggedInTenant() {
 
-        return Optional.ofNullable(IdentityTenantUtil.getTenantDomainFromContext())
+        return Optional.ofNullable(PrivilegedCarbonContext.getThreadLocalCarbonContext().getTenantDomain())
```

Comment:
why we are changing this? IdentityTenantUtil::getTenantDomainFromContext underline handle this.